services/send_user_report.py

```diff
@@ -15,15 +15,15 @@ def get(self):
         self.response.headers['Content-Type'] = 'application/json'
         rate = self.request.get('rate')
         report_rate = int(rate)
-        email_address = "rayolanderos@gmail.com"
+        email_address = "noreply@ut-apt-miniproject-greenteam.appspotmail.com"
```

Comment:
This would only work when deployed under this URL right?

Reply:
yeah :( there must be a way of getting the app alias but nothing i could find easily